database/rrdhost.c

```diff
@@ -741,6 +756,66 @@ struct label *create_label(char *key, char *value, LABEL_SOURCE label_source)
     return result;
 }
 
+struct label *add_label_to_list(struct label *l, char *key, char *value, LABEL_SOURCE label_source)
+{
+    struct label *lab = create_label(key, value, label_source);
+    lab->next = l;
+    return lab;
+}
+
+int list_contains(struct label *head, struct label *check)
```

Comment:
This only a suggestion, Netdata works with different link lists and the name of this function does not say which list we are comparing, but the arguments are clear. My suggestion is to add the work `label` to the name of this function for its name to be more meaningful.

Reply:
Will rename this.